Lib/ssl.py

```diff
@@ -961,8 +961,9 @@ def sendall(self, data, flags=0):
                     self.__class__)
             amount = len(data)
             count = 0
+            view = memoryview(data)
```

Comment:
You should cast the memoryview to a bytes unit, otherwise slicing may work in non-byte units. See e.g. `BufferedIOBase.readinto` at https://github.com/python/cpython/blob/master/Lib/_pyio.py#L695